panel/tests/pane/test_image.py

```diff
@@ -69,11 +69,13 @@ def test_svg_pane(document, comm):
           b'AAAAAAAAAAAAAAAAFBv/EABkRAAEFAAAAAAAAAAAAAAAAAAEAAjFxsf/aAAwDAQ' + \
           b'ACEQMRAD8AA0qs5HvTHQcJdsChioXSbOr/2Q==',
     ico = b'AAABAAEAAgEAAAEAIAA0AAAAFgAAACgAAAACAAAAAgAAAAEAIAAAAAAACAAAAHQ' + \
-          b'SAAB0EgAAAAAAAAAAAAD//////////wAAAAA=')
-
+          b'SAAB0EgAAAAAAAAAAAAD//////////wAAAAA=',
+    webp= b'iVBORw0KGgoAAAANSUhEUgAAAAIAAAABCAYAAAD0In+KAAAAFElEQVQIHQEJAPbA' + \
+          b'WNYYP/h4uMAFL0EwlEn99gAAAAASUVORK5CYIA=='
```

Comment:
```suggestion
    webp= b'UklGRjYAAABXRUJQVlA4ICoAAAIAAQCdASoCAAEAAgA0JYwCdAEO/gLsAP78dui' + \
          b'9+c721FFqJKmvctFAIAA='

```

Try this.

Reply:
Could also update test_img_shape to the following:

``` python
@pytest.mark.parametrize('t', [PNG, JPG, GIF, ICO, WebP], ids=lambda t: t.name.lower())
def test_imgshape(t):
    w, h = t._imgshape(b64decode(twopixel[t.name.lower()]))
    assert w == 2
    assert h == 1
```

Reply:
nice thanks